research/format.markdown

```diff
@@ -0,0 +1,124 @@
+# Research on `format` keyword 
+ Changes in `format` Keyword Behaviour Across all JSON Schema Drafts: Additions, Deletions, and Upgrades.
+
+## Draft 2 to Draft 3
+
+References: [Draft 2 Section 5.20](https://json-schema.org/draft-02/draft-zyp-json-schema-02.txt), [Draft 3 Section 5.23](https://json-schema.org/draft-02/draft-zyp-json-schema-02.txt)
+
+> Behaviour:   
+      >>[Before](https://json-schema.org/draft-02/draft-zyp-json-schema-02.txt): Not required to validate  
+        [After](https://json-schema.org/draft-02/draft-zyp-json-schema-02.txt): MAY validate (but aren’t required to)
```

Comment:
Both references point at Draft 2? Also the before/after here seems exactly the same?

Reply:
I am correcting this..